susemanager-utils/susemanager-sls/salt/srvmonitoring/removejmxprops.sls

```diff
@@ -1,3 +1,8 @@
+remove_{{remove_jmx_props.service}}_jmx_host:
+  cmd.run:
+    - name: sed -i 's/JAVA_OPTS="(.*)-Dcom\.sun\.management\.jmxremote\.host=localhost(.*)"/JAVA_OPTS="\1 \2"/' {{remove_jmx_props.file}}
```

Comment:
This failed with: 

  "sed: -e expression #1, char 90: invalid reference \\2 on `s' command's RHS"

Seems `(` needs to be escaped